### ISSUE TITLE
dry-system 0.16.0 not compatible with dry-configurable 0.11.1

### ISSUE THREAD

**Issue by jswanner** [dry-rb/dry-system#141]
**Describe the bug**

The specs for v0.16.0 this library do not pass when using dry-configurable 0.11.1, 11 examples fail with the following message:

```
Failure/Error: config.settings << _settings[name]

NoMethodError:
  undefined method `settings' for #<Dry::Configurable::Config:0x00007fc757259f38>
  Did you mean?  _settings
```

**To Reproduce**

Clone this repository, check out v0.16.0 (or current master), run bundler, run specs.

**Expected behavior**

The specs for this library should pass.

**Possible solution**

```diff
diff --git a/lib/dry/system/container.rb b/lib/dry/system/container.rb
index cb99fd3..3d72da2 100644
--- a/lib/dry/system/container.rb
+++ b/lib/dry/system/container.rb
@@ -104,7 +104,7 @@ module Dry
         # @api public
         def setting(name, *args, &block)
           super(name, *args, &block)
-          config.settings << _settings[name]
+          config._settings << _settings[name]
           self
         end
```

**Your environment**

- Affects my production application: **NO**, have not upgraded to dry-configurable 0.11.1
- Ruby version: 2.6.5
- OS: MacOS

**Comment by flash-gordon** [comment]
Thank you for reporting

**Comment by solnic** [comment]
I fixed it in 0.17.0 and 0.16.0 was yanked.

**Comment by jswanner** [comment]
Thanks for taking care of this so quickly!
